jest.preset.js

```diff
@@ -15,4 +15,14 @@ module.exports = {
       lines: 50,
     },
   },
+  /* TODO: Update to latest Jest snapshotFormat
+   * By default Nx has kept the older style of Jest Snapshot formats
+   * to prevent breaking of any existing tests with snapshots.
+   * It's recommend you update to the latest format.
+   * You can do this by removing snapshotFormat property
+   * and running tests with --update-snapshot flag.
+   * Example: "nx affected --targets=test --update-snapshot"
```

Comment:
there is no snapshot testing in the repo so I think we can safestly update to the latest format